src/components/music/SongTracker.tsx

```diff
@@ -497,6 +544,23 @@ export const SongTracker = ({
     [activeField]
   );
 
+  const handleWheel = useCallback(
+    (e: any) => {
+      console.log("wheel");
+      if (e.ctrlKey) {
+        if (e.shiftKey) {
+          if (e.deltaY < 0) return transposeSelectedTrackerFields(1,true);
+          if (e.deltaY > 0) return transposeSelectedTrackerFields(-1,true);
+        } else {
+          if (e.deltaY < 0) return transposeSelectedTrackerFields(1,false);
+          if (e.deltaY > 0) return transposeSelectedTrackerFields(-1,false);
+        }
+        return;
+      }
+    },
+    [activeField, selectionRect, pattern]
```

Comment:
You don't need this dependencies for this callback but just `transposeSelectedTrackerFields`. 

I use an eslint rule to warn me about those locally https://github.com/facebook/react/issues/14920

Reply:
Just noticed that once you add `transposeSelectedTrackerFields` as a dependency it'll mean that this functions will be updating on every render. To solve that you'd have to wrap  `transposeSelectedTrackerFields` on its own useCallback with its own dependencies.